modules/glossary/partials/con_glossary.adoc

```diff
@@ -9,23 +9,23 @@
 This section provides an overview of the main terms and expressions used throughout {prod} documentation.
 
 == Naming
-{prod}:: A developer platform for the cloud that provides an in-browser IDE. {prod-short} tools can be extended by adding development services, such as language servers, debug adapters, or editors (IDEs) packaged as containers. 
+{prod}:: A developer platform for the cloud that provides an in-browser editor (IDE). You can enhance {prod-short} tools by adding development services, such as language servers, debug adapters, or IDEs packaged as containers.
 
-Che-Theia:: The default editor component of a {prod-short} workspace. It is a {prod-short}-specific customization of the Eclipse Theia editor. Che-Theia can be extended using Che-Theia plug-ins, which are packaged as containers for ease of deployment and use. Che-Theia plug-ins are usually compatible with Visual Studio extensions.
+Che-Code:: The default editor component of a {prod-short} workspace. It is a {prod-short}-specific customization of the Visual Studio Code editor. You can enhance Che-Code with link:https://marketplace.visualstudio.com/VSCode[Visual Studio Code extensions].
```

Comment:
AFAIK, we do have `che-code` as an editor id value in the docs, but we don't use it as editor name.
According to https://issues.redhat.com/browse/CRW-3098, we call the editor `Visual Studio Code - Open Source`.